frog/imports/ui/TeacherView/Dashboard.jsx

```diff
@@ -20,67 +21,53 @@ const Container = styled.div`
 export class DashboardComp extends Component {
   state: { data: any };
   doc: any;
-  timeout: ?number;
-  mounted: boolean;
+  unmounted: boolean;
+  conn: any;
 
   constructor(props: Object) {
     super(props);
     this.state = { data: null };
   }
 
-  componentDidMount() {
-    this.mounted = true;
```

Comment:
Oh no! Why would you do that ? Isn't it not less easy to read `this.mounted` and `!this.mounted` than `!this.unmounted` and `this.unmounted`